server/api/index.js

```diff
@@ -11,14 +11,6 @@ router.get("/courses", (req, res) => {
   });
 });
 
-// Get 1 Course
-router.get("/courses/:id", (req, res) => {
```

Comment:
this is used in courses container, please, leave it there.

Reply:
sorry about that but api/courses/:id is essential for my issue. Nate gave me a task. I need for the list of lessons. I told Nate. he let me to use. I asked before delete who is using code. but nobody answered me.

Reply:
Currently, removing this code breaks courses functionality because this API is used.

Reply:
There must be some other way to do your task without deleting my api. I am not sure why you need this api for getting lessons. But as Loic said, if you delete this api it will ruin my pages. So it would be great if you can find some other way. Thanks.

Reply:
dont worry islam i fixed your pages